thirdeye-core/src/test/java/ai/startree/thirdeye/mapper/AnomalyFeedbackMapperTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2023 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package ai.startree.thirdeye.mapper;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import ai.startree.thirdeye.spi.api.AnomalyFeedbackApi;
+import ai.startree.thirdeye.spi.datalayer.dto.AnomalyFeedbackDTO;
+import ai.startree.thirdeye.spi.detection.AnomalyFeedbackType;
+import java.sql.Timestamp;
+import java.time.Instant;
+import org.testng.annotations.Test;
+
+public class AnomalyFeedbackMapperTest {
```

Comment:
I think we should not have tests for mappers. 
These tests catch when a mapping is not done anymore but don't catch when a new field is added but not mapped. 

So they tend to be incomplete and poorly maintained. 
At the API level let's just make sure the API contract is respected. HappyPath tests looks good.

Reply:
`These tests catch when a mapping is not done anymore but don't catch when a new field is added but not mapped.` That's correct, the purpose of these tests are for the first case only. It's impossible for unit test itself to understand whether the missing mapping is by intention or mistake. I used to write unit tests for proto converters (similar to this mapping) but I am ok to remove tests here.

Reply:
Yeah I also used to do tons of proto 2 pojo tests and never found such tests useful.

What do you think @suvodeep-pyne @jacksonargo ?

Reply:
I do find the tests useful, especially when debugging the auto-mappers. 

I created a similar unit tests for the dto<->api to verify that the namespace field is properly passed around, because I kept running into cases where the namespace field _wasn't_ mapped for whatever reason.

We can probably use a single test method per Dto/Api instead of method per field, to reduce the boiler-plate code.

Reply:
Preferred smaller tests because I only covered the cases where mapping annotation is used. It's more readable even though there could more boilerplate code, plus it's generally good for CI tools so all errors can be reported in a single run (otherwise test could stop in the middle if there is any mismatch for tests with multiple assertions). 

But if we also want to test the entire dot <-> api then single dto/api method, I am also ok with a single test covering entire dto -> api, and vice versa. But I extracted out the corner case (ignored fields) as a single test because I want the each test case to cover a single behavior.

Reply:
sounds good